README.md

```diff
@@ -13,6 +13,53 @@ This plugin uses [the new Gradle 4.1 and Android Studio 3.0 project setup](https
 * Dart 1: 1.0.x.
 * Dart 2: 1.1.0 and up.
 
+## Before instalation
+
+Before you begin it is important to properly configure your application at https://apps.twitter.com/
+
+It is important to configure the callback URLs so that everything works correctly in your application.
+
+You will have to use the following callback URLs:
+
+Android - twittersdk: //
+
+iOS - twitterkit-CONSUMERKEY: //
+
+FOR MORE INFORMATION READ:
+https://developer.twitter.com/en/docs/basics/developer-portal/guides/callback-urls.html
+
+[Complete Guide on ](https://github.com/twitter/twitter-kit-ios/wiki/Installation)
+
+
+
+
+### Configure Info.Plist
+Twitter Kit looks for a URL scheme in the format twitterkit-<consumerKey>, where consumerKey is your application's Twitter API key, e.g. twitterkit-dwLf79lNQfsJ.
+
+You can find your consumer key in the Twitter app dashboard.
```

Comment:
You can probably append the link to the dashboard here. https://developer.twitter.com/en/dashboard